hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeExtension.java

```diff
@@ -186,4 +186,9 @@ public void destroy() {
         logger.info("Destroying node NodeExtension.");
     }
 
+    @Override
+    public void beforeJoin() {
+        logger.info("Before join check.");
```

Comment:
What's the use of this log? If this method is doing nothing then no need to log anything. I think you should remove it or at least make it `FINEST`.